example/apps/test_security/management/commands/test_command_with_response.py

```diff
@@ -0,0 +1,9 @@
+from django.core.management.base import BaseCommand
+
+from security import requests
+
+
+class Command(BaseCommand):
+
+    def handle(self, **options):
+        requests.post('http://test.cz/test')
```

Comment:
You shouldn't try to access live servers in tests no matter what. Why not `http://localhost`?

Reply:
but the tests should mock requests